gravitee-cockpit-connectors-ws/src/main/java/io/gravitee/cockpit/connectors/ws/channel/ClientChannel.java

```diff
@@ -202,8 +202,7 @@ void listen() {
                         webSocket.close((short) 400, "Bad incoming content");
                     }
                 } catch (Exception e) {
-                    log.warn(String.format("An error occurred when trying to decode incoming content [%s]. Closing Socket.", incoming), e);
-                    webSocket.close((short) 500, "Unexpected error");
+                    log.info(String.format("An error occurred when trying to decode incoming content [%s]. Ignore message.", incoming), e);
```

Comment:
Maybe we could use `error` level 🤔 

```suggestion
                    log.error(String.format("An error occurred when trying to decode incoming content [%s]. Ignore message.", incoming), e);
```

Reply:
As discussed with @jhaeyaert, we think error level is enough because we don't want the prod logs of our customers to contain errors only because there is a new version of Cockpit. From the APIM / AM point of view, this is not really an error, just a feature they can not benefit of before upgrading their installation.